temba/contacts/tasks.py

```diff
@@ -117,3 +123,100 @@ def check_elasticsearch_lag():
                 return False
 
     return True
+
+
+@nonoverlapping_task(track_started=True, name="block_deactivated_contacts_task")
+def block_deactivated_contacts_task():
+    email_subject = f"{timezone.now().strftime('%B %d, %Y')} - list of deactivated phone numbers."
+    email_text = """
```

Comment:
Actually, one last thing. Other parts of the system are using an HTML template for the email (https://github.com/greatnonprofits-nfp/rapidpro/blob/main/templates/notifications/email/export_finished.results.html), I think would be good to have it in a template as well.